tools/Status/Status.py

```diff
@@ -506,9 +553,35 @@ def render_status(
         "See `spectre -h` for its path."
     ),
 )
-def status_command(refresh_rate, **kwargs):
+@click.option(
+    "--columns",
+    "-c",
+    "input_columns",
```

Comment:
`standard_columns` (`input` is not a very useful name)

Reply:
Actually I think `input` here is more descriptive than `standard`. And also more accurate. These are columns that the user passes to the command as "inputs". There's no reason these passed in columns are "standard" by any metric (unless they use the defaults)

Reply:
you can't and shouldn't prefix every option or function argument with "input" just because it's a "user input".

I'm not attached to "standard". I chose that name to distinguish from the "extra" columns that each executable adds ("standard" = same for each exec). Probably just "columns" is fine too (but a bit misleading because more columns are added per exec).

Reply:
How about "user_columns"

Reply:
The intention is to distinguish from the exec-specific columns. I have no better idea than "standard" or "generic". Just plain "columns" is also fine.

Reply:
`columns_to_use` seems no better than just `columns` 😅

Reply:
I feel like I'm getting too pedantic here though so if you prefer that name for some reason then that's fine with me.

Reply:
I prefer the name that gets this merged the fastest lol. I think `columns_to_use` is good enough for this.

Reply:
Change it to `columns` then. It's public API people set in a config file and without a reason for making it different from the `--columns` option name it's unnecessarily confusing.

Reply:
done